LondonBritishLibrary/orient/BLorient687.xml

```diff
@@ -37,6 +42,507 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.10">
+                                <locus from="61ra" to="62rc"/>
+                                <title type="complete" ref="LIT3159Cyprian"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.11">
+                                <locus from="62va" to="68rc"/>
+                                <title type="complete" ref="LIT3160Sergius"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.12">
+                                <locus from="68va" to="69vc"/>
+                                <title type="complete" ref="LIT3161Pelagia"/>
+                                <textLang mainLang="gez"/>
+                                <note>For <date ref="ethiocal:Teqemt11">11 Ṭǝqǝmt</date>.</note>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.13">
+                                <locus from="70ra" to="71rb"/>
+                                <title type="complete" ref="LIT3162Phileas"/>
+                                <textLang mainLang="gez"/>
+                                <note>For <date ref="ethiocal:Teqemt17">17 Ṭǝqǝmt</date>.</note>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.14">
+                                <locus from="71va" to="74rc"/>
+                                <title type="complete" ref="LIT3163Romanus"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.15">
+                                <locus from="74va" to="79rc"/>
+                                <title type="complete" ref="LIT3164JohnDailami"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.16">
+                                <locus from="79va" to="81rc"/>
+                                <title type="complete" ref="LIT6512Zenobius"/>
+                                <textLang mainLang="gez"/>
+                                <note>For <date ref="ethiocal:Hedar6">6 Ḫǝdār</date>.</note>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.17">
+                                <locus from="81va" to="91vc"/>
+                                <title type="complete">Life and Miracles of St George</title>
```

Comment:
this is CAe 1455

Reply:
Sorry I guess here too we need a new ID for the dossier of St George if not available